doc/content/getting-started/migrating/migrating-from-v2/migrate-using-migration-tool/migrate-active-session.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Migrating Active Sessions"
+description: ""
+weight: 2
+new_in_version: "3.12.0"
+distributions: ["Cloud","Dedicated Cloud","AWS Launcher","Enterprise","Open Source"]
+---
+
+Starting from {{% tts %}} version `3.12.0`, it is possible to migrate end devices together with their active sessions using the `ttn-lw-migrate` tool. This section explains how to export one or more of your end devices from {{% ttnv2 %}} to a [JSON file]({{< ref "/getting-started/migrating/device-json" >}}), with persisting their active sessions.
+
+<!--more-->
+
+{{< warning >}} Active device sessions can be migrated via Packet Broker only from **The Things Industries V2 (SaaS)** to **{{% tts %}} Cloud**, and even this is achievable only on a customer request. Contact [our support](mailto:support@thethingsindustries.com) for more information. 
```

Comment:
```suggestion
{{< warning >}} Active device sessions can be migrated via Packet Broker only from **The Things Industries V2 (SaaS)** to **{{% tts %}} Cloud**, and this is achievable only on a customer request. Contact [The Things Industries support](mailto:support@thethingsindustries.com) for more information. 
```

Why the many distributions in the frontmatter then?

Reply:
Yeah, this seems incorrect. 

If I understand well, session migration is available for all distributions, only most of them require gateway to be migrated to TTS as well. Should we mark this as a feature available for all distributions, or only for Cloud (since we discourage migrating gateways from TTN)?